canonical-web-and-design/maas-ui
# fix(legacy): fix discovery enabled value

## Done

- Use the correct value for enabling discovery.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the dashboard (click on the header logo).
- Go to the Configuration tab.
- Toggle 'Discovery enabled' to on.
- An error should not appear.
- Refresh the page and go back to the Configuration tab. The toggle should still be on.

## Fixes

Fixes: #2602.
